packages/spatial/src/renderer/functions/performanceHooks.ts

```diff
@@ -27,24 +27,46 @@ import { State, useHookstate, useMutableState } from '@etherealengine/hyperflux'
 import { useEffect } from 'react'
 import { PerformanceState } from '../PerformanceState'
 
-export const usePerformanceTier = (): State<number> => {
+export const useGPUPerformanceTier = (): State<number> => {
   const performanceState = useMutableState(PerformanceState)
-  const performanceTier = useHookstate(performanceState.tier.value)
+  const performanceTier = useHookstate(performanceState.gpuTier.value)
 
   useEffect(() => {
-    performanceTier.set(performanceState.tier.value)
-  }, [performanceState.tier])
+    performanceTier.set(performanceState.gpuTier.value)
+  }, [performanceState.gpuTier])
 
   return performanceTier
 }
 
-export const usePerformanceOffset = (): State<number> => {
+export const useCPUPerformanceTier = (): State<number> => {
```

Comment:
i think these hooks are a bit overengineered, they can be as simple as `useHookstate(getMutableState(PerformanceState).cpuTier)`, unless this is in preperation for more sophisticated and configurable heuristic functions, in which case these still seem premature